examples/device/ep/tests/elastic/elastic.py

```
@@ -664,14 +668,21 @@ def main():
 
     args = parser.parse_args()
 
+    if args.kineto:
+        torch.cuda.set_device(SINGLE_PROCESS_WORKER_RANK)
+        if not kineto_cuda_available():
+            raise SystemExit(
+                "Kineto profiling was requested but is unavailable"
+            )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)